src/cljc/matcher_combinators/core.cljc

```diff
@@ -143,6 +143,16 @@
       issue
       (compare-maps expected actual model/->Unexpected false))))
 
+(defrecord EqualsRecord [expected]
```

Comment:
an `EmbedsRecord` probably makes sense as well

Reply:
Seems like records always consider inexistent properties as `nil`, so it will behave like `equals`